scripts/measures/enrich-measures-data.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs');
+const path = require('path');
+const parse = require('csv-parse/lib/sync');
+
+const aciRelations = require('../../util/measures/aci-measure-relations.json');
+const cpcPlusGroups = require('../../util/measures/cpc+-measure-groups.json');
+
+const qpp = fs.readFileSync(path.join(__dirname, '../../staging/measures-data.json'), 'utf8');
+fs.writeFileSync(path.join(__dirname, '../../measures/measures-data.json'), enrichMeasures(JSON.parse(qpp)));
+
+function enrichMeasures(measures) {
+  enrichACIMeasures(measures);
+  enrichCPCPlusMeasures(measures);
+  enrichAddMeasuresSpecification(measures);
+  return JSON.stringify(measures, null, 2);
+};
+
+/**
+ * Will add extra metadata to ACI measure that are not directly available
+ * in machine inferable format at https://qpp.cms.gov/api/v1/aci_measures
+ * After this function executes, an ACI measure will have reporting category and substitutes.
+ *  - substitutes: contains other measures that surrogates of a given measure.
+ *  - reportingCategory: corresponds to the measure performance category
+ * @param measures - the measures to enrich
+ */
+function enrichACIMeasures(measures) {
+  // add extra ACI metadata to ACI measure
+  measures
+    .filter(measure => measure.category === 'aci')
+    .forEach(measure => {
+      // find the relation and populate reporting category and substitutions
+      var aciRelation = aciRelations[measure.measureId];
+      if (aciRelation) {
+        measure.reportingCategory = aciRelation.reportingCategory;
```

Comment:
Do these fields need to be added to the schema?

Reply:
The `reportingCategory` and `substitutes` attributes have already been added to the `aciMeasures` schema in a previous commit https://github.com/CMSgov/qpp-measures-data/blob/65864866f828c349ef41e55f9b42e69d457492f9/measures/measures-schema.yaml#L60

The change here related to ACI is moving the ACI enriching function to a single script with other measure enriching functions.